google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -69,7 +69,7 @@ def set_run(self, run: str):
             schema_title=constants.SYSTEM_METRICS,
             schema_version=constants.SCHEMA_VERSIONS[constants.SYSTEM_METRICS],
         )
-        run_execution.add_artifact(artifact=metrics_artifact.resource_name, input=False)
+        run_execution.add_artifact(artifact_resource_name=metrics_artifact.resource_name, input=False)
```

Comment:
nit: artifact_name

Reply:
I'd prefer artifact_resource_name here as `resource.name` method would actually return the resource ID instead of the full resource name, so artifact_name could be misleading and confuse the user whether we are expecting a full resource name or just the resource ID here.

Reply:
Example of other places:
https://github.com/googleapis/python-aiplatform/blob/3082f97837a4f2c033c9db4ad752f51419ef17cf/google/cloud/aiplatform/datasets/dataset.py#L49
https://github.com/googleapis/python-aiplatform/blob/3082f97837a4f2c033c9db4ad752f51419ef17cf/google/cloud/aiplatform/datasets/dataset.py#L58

no strong opinion, fine with either, up to you